src/main/java/frc/robot/RobotContainer.java

```diff
@@ -0,0 +1,240 @@
+import frc.robot.subsystems.shooter.ShooterCMD;
+import frc.robot.subsystems.shooter.ShooterSS;
+import frc.robot.subsystems.spinner.SpinnerSS;
+import frc.robot.utilities.DashboardController;
+import frc.robot.utilities.TrigonXboxController;
+import frc.robot.vision.Target;
+import frc.robot.vision.limelights.PitcherLimelight;
+import io.github.oblarg.oblog.Logger;
+
+public class RobotContainer {
+    private final RobotA robotConstants;
+    private final SubsystemContainerA subsystemContainer;
+    private final TrigonXboxController driverXboxController;
+    private final TrigonXboxController overrideXboxController;
+    private final DashboardController dashboardController;
+    private final PitcherLimelight limelight;
+
+    private ShooterCMD shooterCMD;
+    private CalibrateShooterKfCMD calibrateShooterKfCMD;
+    private GenericCalibrateKF calibrateLoaderKfCMD;
+    private SupplierFieldDriveCMD supplierFieldDriveCMD;
+    private TurnToTargetCMD turnToTargetCMD;
+    private TurnAndPositionToTargetCMD turnAndPositionToTargetCMD;
+    private ToggleMotorsModeCMD toggleMotorsModeCMD;
+
+    private TrigonSwerveControllerCMDGP motionTest;
+    private IntakeOpenerCMD intakeCMD;
+
+    private ShootCMDGP shootCMDGP;
+    private ShootWithPitcherCMDGP ShootWithPitcherCMDGP;
+    private CollectCMDGP collectCMDGP;
+
+    private RunWhenDisabledCommand resetDirection;
+
+    /**
+     * Add classes here
+     */
+    public RobotContainer() {
+
+        Logger.configureLoggingAndConfig(this, true);
+        robotConstants = new RobotA();
+        subsystemContainer = new SubsystemContainerA();
+        dashboardController = new DashboardController();
+        driverXboxController = new TrigonXboxController(0);
+        overrideXboxController = new TrigonXboxController(1);
+        limelight = new PitcherLimelight(robotConstants.extendedLimelightConstants,
+                robotConstants.retractedLimelightConstants, subsystemContainer.PITCHER_SS);
+
+        initializeCommands();
+        bindDriverCommands();
+        bindOverrideCommands();
+        setShuffleBoard();
+
+        SmartDashboard.putData("Shooter Command", shooterCMD);
+        SmartDashboard.putData("CalibrateShooterKfCMD", calibrateShooterKfCMD);
+        SmartDashboard.putData("CalibrateLoaderKfCMD", calibrateLoaderKfCMD);
+        SmartDashboard.putData("TurnToTargetCMD", turnToTargetCMD);
+        SmartDashboard.putData("TurnAndPositionToTargetCMD", turnAndPositionToTargetCMD);
+        SmartDashboard.putData("TrigonSwerveControllerCMDGP", motionTest);
+        SmartDashboard.putNumber("Shooter/Desired Velocity", subsystemContainer.SHOOTER_SS.getVelocityRPM());
+
+        Logger.configureLogging(subsystemContainer.DRIVETRAIN_SS);
+        limelight.startVision(Target.PowerPort);
+    }
+
+    /**
+     * Initializes all commands.
+     */
+    public void initializeCommands() {
+        SmartDashboard.putNumber("Shooter/Desired Velocity", 0);
```

Comment:
It's not a command that you're initializing, so don't put it here

Reply:
this was required because that value is given to the command in the constructor and if you attempt to get a value without sending it first weird and annoying things happen